public/app/core/utils/explore.ts

```diff
@@ -395,3 +385,11 @@ export const getQueryKeys = (queries: DataQuery[], datasourceInstance: DataSourc
 
   return queryKeys;
 };
+
+export const getTimeRange = (timeZone: TimeZone, rawRange: RawTimeRange): TimeRange => {
```

Comment:
Should this be here or moved to selectors? I put it here since the function doesn't take a state object as argument like ExploreItemState, instead it takes the actual "properties" needed.

Reply:
This is a legit place for such utils